docs/quickstart.rst

```diff
@@ -129,6 +129,38 @@ certain number of old archives::
     borg prune -v $REPOSITORY --prefix '{hostname}-' \
         --keep-daily=7 --keep-weekly=4 --keep-monthly=6
 
+Pitfalls with shell variables and environment variables
+-------------------------------------------------------
+
+This applies to all environment variables you want borg to see, not just
+``BORG_PASSPHRASE``. The short explanation is: always ``export`` your variable,
+and use single quotes if you're unsure of the details of your shell's expansion
+behavior. eg::
+
+    export BORG_PASSPHRASE='your complicated & passphrase';
```

Comment:
> complicated & long

Done

> the ; at the end is not needed, is it?

Probably not, but just trying to be complete, for those who aren't proficient with shell (that is: safe for someone who sees the "tl;dr" and copy/pastes)